release_tester/test_driver.py

```diff
@@ -589,3 +598,49 @@ def run_license_manager_tests(
                     result["messages"].append(one_result.message)
             results.append(result)
         return results
+
+    def run_all_test_suites(self, versions):
+        """Run all independent test suites"""
+        results = []
+        results.extend(self.run_conflict_tests(versions, True))
+        results.extend(self.run_conflict_tests(versions, False))
+        results.extend(self.run_license_manager_tests(versions))
+        return results
+
+    def run_test_suites(self, versions: list, include_suites=(), exclude_suites=()):
+        suite_classes=[]
+        if len(include_suites)==0 and len(exclude_suites)==0:
+            return self.run_all_test_suites(versions)
+        elif len(include_suites)>0 and len(exclude_suites)==0:
```

Comment:
everywhere else in this PR you have been reformatting ==/> to contain blanks around them? ;-)

Reply:
I had to correct this part manually. I don't know why black did not take care of this.